data-collection/data-collection/Rich Popup/Attachments/RichPopupStagedPhotoAttachment.swift

```diff
@@ -27,13 +27,30 @@ class RichPopupStagedPhotoAttachment: RichPopupStagedAttachment {
     
     let image: UIImage
     
+    var nameAsJPEG: String? {
+        
+        guard var jpegName = name?.trimmingCharacters(in: .whitespacesAndNewlines) else { return nil }
+        
+        let fileExtension = NSString(string: jpegName).pathExtension
```

Comment:
I would suggesting bridging `jpegName` so as not to create a new `NSString` object.

```suggestion
        let fileExtension = (jpegName as NSString).pathExtension
```